docs/_interface/Private-Network.md

```diff
@@ -0,0 +1,291 @@
+
+#### Clique Example
+
+This is an example of a genesis.json file for a proof-of-authority network. The `config`
+section ensures that all known protocol changes are available and configures the 'clique'
+engine to be used for consensus.
+
+Note that the initial signer set must be configured through the `extradata` field. This
+field is required for clique to work.
+
+First create the signer account keys using the [geth account](./managing-your-accounts)
+command (run this command multiple times to create more than one signer key).
+
+```shell
+geth account new --datadir data
+```
+
+Take note of the Ethereum address printed by this command.
+
+To create the initial extradata for your network, collect the signer addresses and
+hex-encode `extradata` as the concatenation of 32 zero bytes, all signer addresses, and 65
+further zero bytes. In the example below, `extradata` contains a single initial signer
+address, `0x7df9a875a174b3bc565e6424a0050ebc1b2d1d82`.
+
+```json
+{
+  "config": {
+    "chainId": 15,
+    "homesteadBlock": 0,
+    "eip150Block": 0,
+    "eip155Block": 0,
+    "eip158Block": 0,
+    "byzantiumBlock": 0,
+    "constantinopleBlock": 0,
+    "petersburgBlock": 0
+    "clique": {
+      "period": 5,
+      "epoch": 30000
+    }
+  },
+  "difficulty": "1",
+  "gasLimit": "8000000",
+  "extradata": "0x00000000000000000000000000000000000000000000000000000000000000007df9a875a174b3bc565e6424a0050ebc1b2d1d820000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000",
+  "alloc": {
+    "7df9a875a174b3bc565e6424a0050ebc1b2d1d82": { "balance": "300000" },
+    "f41c74c9ae680c1aa78f42e5647a62f353b7bdde": { "balance": "400000" }
+  }
+}
+```
+
+#### Ethash Example
+
+Since ethash is the default consensus algorithm, no additional parameters need to be
+configured in order to use it. You can influence the initial mining difficulty using the
+`difficulty` parameter, but note that the difficulty adjustment algorithm will quickly
+adapt to the amount of mining resources you deploy on the chain.
+
+```json
+{
+  "config": {
+    "chainId": 15,
+    "homesteadBlock": 0,
+    "eip150Block": 0,
+    "eip155Block": 0,
+    "eip158Block": 0,
+    "byzantiumBlock": 0,
+    "constantinopleBlock": 0,
+    "petersburgBlock": 0,
+    "ethash": {}
+  },
+  "difficulty": "1",
+  "gasLimit": "8000000",
+  "alloc": {
+    "7df9a875a174b3bc565e6424a0050ebc1b2d1d82": { "balance": "300000" },
+    "f41c74c9ae680c1aa78f42e5647a62f353b7bdde": { "balance": "400000" }
+  }
+}
+```
+
+### Initializing the Geth Database
+
+To create a blockchain node that uses this genesis block, run the following command. This
+imports and set the canonical genesis block for your chain.
+
+```shell
+geth init --datadir data genesis.json
+```
+
+Future runs of geth using this data directory will use the genesis block you have defined.
+
+```shell
+geth --datadir data --networkid 15
```

Comment:
Why do I have to specify the network id every time if I already did so in the genesis block?

Reply:
Chain ID and network ID are two different things. This is actually really annoying and we could fix it by auto-setting the `--networkid` to chain ID, but it's not how geth behaves right now.